lib/roo_on_rails/railties/logging.rb

```diff
@@ -8,7 +8,8 @@ class Logging < Rails::Railtie
       end
 
       initializer 'roo_on_rails.logging.after', after: :initialize_logger do
-        Rails.logger.set_log_level
+        log_level = Rails.configuration.log_level
+        Rails.logger.set_log_level(default: log_level)
```

Comment:
If `log_level` is nil, what is the behaviour here? Will `set_log_level` attempt to call `upcase` on it and the app will blow up?

Reply:
Rails will raise an error if `log_level` is set to something invalid (and it defaults to debug) so it cannot end up being nil at app boot. But yes, there should be a test here to check that the validation of this doesn't occur later on in the boot sequence.

Reply:
I checked, and the setter itself will raise an error if you set it to something invalid. I've added a comment to explain this as I don't want to write a test for Rails behaviour.